### PR TITLE
fix multiselect dropdowns in variant creator

### PR DESCRIPTION
**Pull request by SektorDV**
I want to merge this change because it fixes the issues present when there's more than one multiselect on the variant creator page.

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://qa.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3617 Dropdowns bug](https://app.clickup.com/t/2549495/SALEOR-3617)

File: src/products/components/ProductVariantCreatorPage/ProductVariantCreatorValues.tsx [hunk 1]
```diff
@@ -10,6 +10,8 @@ import { getById } from "@saleor/orders/components/OrderReturnPage/utils";
 import { ProductDetails_product_productType_variantAttributes } from "@saleor/products/types/ProductDetails";
 import { SearchAttributeValues_attribute_choices_edges_node } from "@saleor/searches/types/SearchAttributeValues";
 import { FetchMoreProps } from "@saleor/types";
+// eslint-disable-next-line no-restricted-imports
+import _ from "lodash";
```
Comment (dominik-zeglen) [inline src/products/components/ProductVariantCreatorPage/ProductVariantCreatorValues.tsx:14]:
```suggestion
import isEqual from "lodash/isEqual";
```

**Review by dominik-zeglen — CHANGES_REQUESTED** [review]
I don't think it's the right approach - passing attribute values if the attribute id matches the one used by query and empty array if not makes more sense (and it's easier to understand what is actually happening in there) than using memoization with deep compare + onBlur handler

**Comment by jwm0** [comment]
> I don't think it's the right approach - passing attribute values if the attribute id matches the one used by query and empty array if not makes more sense (and it's easier to understand what is actually happening in there) than using memoization with deep compare + onBlur handler

In general I believe the issue is not with the component itself but rather the way we're it. Single root level state for multiple selects doesn't work. Having it as a single state is fine but it should be persisted inside some key to maintain the cache and have a clear seperation. (using the select `name` as a key should be possible)

**Review by jwm0 — CHANGES_REQUESTED** [review]
See my above comment

**Pull request closed without merging**